lib/ansible/modules/windows/win_service.py

```diff
@@ -113,6 +113,20 @@
     - A newly created service will default to C(LocalSystem).
     type: str
     version_added: '2.3'
+  recovery_reset_interval:
+    description:
+      - Reset failure counter after this number of seconds.
+    type: int
+    default: 0
+    version_added: '2.10'
+  recovery_actions:
+    description:
+      - A list representing the recovery actions you want to set (3).
+      - First element of the list is the first failure actions/delay...
+      - Third element is for the subsequent failures.
+    type: list
```

Comment:
This takes a list but each entry is a dict. We should define the options in each dict. We can do this by adding

```
type: list
suboptions:
  action:
    description:
    - The description of the action
    choices:
    - restart
    - reboot
    - run
    required: yes
    type: str
  delay:
    description:
    - The description of delay
    type: int
    required: yes
```

The values of `suboptions` is just like the normal module options at the root level, so you can set things like `description`, `type`, `choices`, `default`, `required`, `version_added` and so forth. Doing this allows people to see what they can specify for each action and what it actually means rather than trying to decipher the examples alone.